py/picca/delta_extraction/utils_pk1d.py

```diff
@@ -16,6 +16,7 @@
 # create logger
 module_logger = logging.getLogger(__name__)
 
+
 def exp_diff(hdul, log_lambda):
```

Comment:
I do not understand if you merged my 'exp_diff_desi' with the old exp_diff function. If the exp_diff_desi is not added, please add it as it is, and I will make the appropriate changes to adapt it to the data.

Reply:
no, it's not added yet. The issue is not only the script, but also the interfacing with the data model (iirc currently the diff spectra for desi Pk1d data are just fixed to zero), but we could should change this

Reply:
the exp_diff_desi is now added as well, needs to be tested, though